### PR TITLE
To merge in global-workflow build option

### PR DESCRIPTION
**Pull request by jkbk2004**
This is initial PR to provide the global-workflow capability in the ufs-mrweather-app repo. 

1. A shell script is added to check out and build global-workflow. 
2. Basic instructions are given in README.
3. Tested ok on hera until generating rocoto xml file to set forecast experiment.
4. Detail steps to launch out the forecast experiment will be followed in next PRs.

### CODE REVIEW

**Review by ulmononian — APPROVED** [review]
Builds successfully on Hera and Orion. Case build/set-up functional using arbitrary C384, coupled ATM test; rocoto XML & Cron files generate successfully.

File: README.md [hunk 1]
```diff
@@ -7,4 +7,533 @@ includes a full workflow, with pre-processing (preparation of inputs),
...
-https://github.com/ufs-community/ufs-mrweather-app/wiki
\ No newline at end of file
+https://github.com/ufs-community/ufs-mrweather-app/wiki
+
+Getting started
+
+1. Clone ufs-mrweather-app and check out global-workflow
+
+```
+git clone -b feature/global_workflow https://github.com/jkbk2004/ufs-mrweather-app
```
Comment (ligiabernardet) [inline README.md:17]: It seems this should not point to someone's fork. The user should clone the master branch of the ufs-mrweather-app

**Review by ligiabernardet — CHANGES_REQUESTED** [review]
I can't see a reason for keeping file Externals.cfg.v1.0 in the repo. If someone needs to know what Externals.cfg looked like for the v1 release, they can look at the v1 tag. I suggest deleting this file.

File: README.md (hunk 1 above)
Comment (jkbk2004) [reply README.md]: @ligiabernardet corrected the url

**Review by ligiabernardet — APPROVED** [review]

**Pull request merged**